src/models/student.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const subject = mongoose.Schema({
+  title: {
+    type: String,
+    enum: ['Math', 'Physics', 'Biology', 'History', 'Programming'],
+    default: 'Math',
+    required: true,
+  },
+});
+
+const student = mongoose.schema({
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  password_hash: {
```

Comment:
let's follow the same naming convention, if you started with camelCase you should continue with it instead of using sanek_case